net/aria2/files/aria2.init

```diff
@@ -188,10 +188,10 @@ aria2_start() {
 	append_setting "input-file=${session_file}"
 	append_setting "save-session=${session_file}"
 
-	if [ -z "$enable_logging" ]; then
+	if [ -z "$enable_log" ]; then
```

Comment:
this option rename will break existing configs, causing unneeded surprises.  please do not do it.

Reply:
@sotux 

> fix log option according to the luci-app-aria2 setting

If your reasoning was the LuCI app has a different options than aria itself, then you should fix the LuCI app, not the main package.

Reply:
OK. Then I will fix only typo in this package and fix the log option in the luci app. Thanks.